Windows/quickstart-csharp/Program.cs

```diff
@@ -30,7 +30,7 @@ class Program
                 }
                 else
                 {
-                    Console.WriteLine($"We recognized: {result.RecognizedText}");
```

Comment:
>RecognizedText [](start = 63, length = 14)

aren't there other sample accessing RecognizedText and need to get updated??

Reply:
All samples, except this quickstrart_csharp and quickstrat_cpp, will be merged from master, which are from samples.zip. Only these two orphan files are not tested with build. We need to include them into some build process.